avm/res/api-management/service/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -150,6 +169,24 @@ module testDeployment '../../../main.bicep' = [
           ]
         }
       ]
+      loggers: [
+        {
+          name: 'logger'
+          loggerType: 'applicationInsights'
+          isBuffered: false
+          description: 'Logger to Azure Application Insights'
+          credentials: {
+            instrumentationKey: nestedDependencies.outputs.appInsightsInstrumentationKey
+          }
+          resourceId: nestedDependencies.outputs.appInsightsResourceId
+        }
+      ]
+      managedIdentities: {
+        systemAssigned: true
+        userAssignedResourceIds: [
+          nestedDependencies.outputs.managedIdentityResourceId
+        ]
+      }
```

Comment:
Required for WAF alignment?

Reply:
yup, required for [Azure.APIM.ManagedIdentity](https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.APIM.ManagedIdentity)